manifests/params.pp

```diff
@@ -160,6 +160,13 @@
   $mariadb10_includepkgs = absent
   $mariadb10_exclude = absent
 
+  # MongoDB
+  $mongodb_url = "https://repo.mongodb.org/yum/redhat/${::operatingsystemmajrelease}/mongodb-org/3.2/x86_64/"
```

Comment:
The Mongo docs say they only support x86_64, but I don't see any harm in doing it just for fun.